common/v2/providers/ScreenLockProvider/ScreenLockProvider.tsx

```diff
@@ -2,10 +2,17 @@ import React, { Component } from 'react';
 import { withRouter, RouteComponentProps } from 'react-router-dom';
 import CryptoJS, { SHA256, AES } from 'crypto-js';
 
-import * as localCacheService from 'v2/services/LocalCache/LocalCache';
-import * as screenLockService from 'v2/services/ScreenLockSettings/ScreenLockSettings';
-import * as settingsService from 'v2/services/Settings/Settings';
 import { translateRaw } from 'translations';
+import {
+  readAllSettings,
+  getCache,
+  setEncryptedCache,
+  destroyCache,
+  getEncryptedCache,
+  setCache,
+  destroyEncryptedCache
+} from 'v2/services/Store';
+import * as screenLockService from 'v2/services/Store/ScreenLock/ScreenLockSettings';
```

Comment:
Consider importing the functions directly here, similar to how it's done in other files.

Reply:
Done